grails-app/assets/javascripts/meriplan.js

```diff
@@ -908,6 +919,119 @@ function DetailsViewModel(o, project, budgetHeaders, risks, allServices, selecte
             healthCheckUrl:config.healthCheckUrl
         });
 };
+
+function ChangedDetailsViewModel(o, project, budgetHeaders, risks, allServices, selectedTargetMeasures, config) {
```

Comment:
This looks like a copy/paste?  It would be better to _.extend the DetailsViewModel then add any extra functionality you need here.